### DISCUSSION TITLE
Adosearch - advanced search across multiple models

### DISCUSSION THREAD

**Discussion by Obapelumi** [adonisjs/core#3774] [Show and tell]
So excited to finally release [adosearch](https://github.com/Obapelumi/adosearch) 

Adosearch is an [Adonis.js Query Scope](https://docs.adonisjs.com/guides/models/query-scopes#document) that makes advanced search across multiple models trivial to implement.
Inspired by the [sofa/eloquence-base](https://github.com/jarektkaczyk/eloquence/wiki/Builder-searchable-and-more) laravel package.

**Comment by thetutlage** [comment]
Great. Can you please add it to the https://github.com/adonisjs-community/awesome-adonisjs repo?

**Reply by Obapelumi** [reply]
Yes, of course

**Reply by thisisrifai** [reply]
Wish that you add support for adonis v6, @thetutlage I realy want to contribute in adonisjs community but I'm self learner and I think I'm not ready yet if you add an article in the new adonisjs blog showing how to develop adonisjs v6 packages will be a holy gift, for me and the community.
